backend/queries/mentorPairs.js

```diff
@@ -13,7 +13,7 @@ const getMentorPairByVolunteerId = async (id) => {
 const getMentorPairByFellowId = async (id) => {
     const selectQuery = `
         SELECT * 
-        FROM mentor_pairs INNER JOIN volunteers ON mentee = v_id
+        FROM mentor_pairs INNER JOIN volunteers ON mentor = v_id
```

Comment:
isnt the fellow the mentee? i think in getMentorPairByVolunteerId above, you change ON mentee = f_id to ON mentor

Reply:
You’re right, this has been corrected in the next PR